grammars/shell-unix-bash.cson

```diff
@@ -503,6 +503,50 @@
         ]
       }
       {
+        'begin': '(<<)-("|\'|)(BASH|SHELL)\\2'
```

Comment:
Not sure if `BASH` should be included here anymore since bash is a subset of shell files.